app/download/views.py

```diff
@@ -13,7 +13,7 @@ def download_document(service_id, document_id):
         return jsonify(error='Missing decryption key'), 400
 
     filename = request.args.get('filename')
-    sending_method = request.args.get('sending_method')
+    sending_method = request.args.get('sending_method', "link")
```

Comment:
`None` is treated as "link" later on. Making it explicit here allows the url returned by url_for( 'download.download_document_b64',...) to be simpler.